In Debian, Postfix's smtp(8) uses a different name to avoid naming conflicts.
Need to point lmtp(8) to the right file.  See #920356.
--- a/man/man8/lmtp.8
+++ b/man/man8/lmtp.8
@@ -1 +1 @@
-.so man8/smtp.8
+.so man8/smtp.8postfix
